set.mm

```diff
@@ -149685,11 +149685,11 @@ seq M ( .+ , F ) ) $=
     seradd.2 $e |- ( ( ph /\ k e. ( M ... N ) ) -> ( F ` k ) e. CC ) $.
     seradd.3 $e |- ( ( ph /\ k e. ( M ... N ) ) -> ( G ` k ) e. CC ) $.
     seradd.4 $e |- ( ( ph /\ k e. ( M ... N ) ) ->
-                     ( H ` k ) = ( ( F ` k ) + ( G ` k ) ) ) $.
+                                     ( H ` k ) = ( ( F ` k ) + ( G ` k ) ) ) $.
     $( The sum of two infinite series.  (Contributed by NM, 17-Mar-2005.)
```

Comment:
I think there is no convention/agreement that a (follow up) line should always be right aligned. I would prefer the following line break/indentation:

seradd.4 $e |- ( ( ph /\ k e. ( M ... N ) )
                        -> ( H ` k ) = ( ( F ` k ) + ( G ` k ) ) ) $.

Reply:
I prefer the original indentation (which I think is also mmj2 style). Right alignment is way too much work to keep in sync.

Reply:
Ok. Reverted these alignments (the ones where the mmj2-style indentation was followed)